plugin-catalog/src/components/plugins/List.tsx

```diff
@@ -0,0 +1,289 @@
+              margin: '1rem',
+              alignSelf: 'flex-start',
+            }}
+            component="img"
+          />
+          <Box display="flex" alignItems="center" justifyContent="space-around" marginRight="10px">
+            {(props.official || props.repository.official) && (
+              <Tooltip title="Official Chart">
+                <Icon
+                  icon="mdi:star-circle"
+                  style={{
+                    marginLeft: '0.5em',
+                    fontSize: '22px',
+                  }}
+                />
+              </Tooltip>
+            )}
+            {props.repository.verified_publisher && (
+              <Tooltip title="Verified Publisher">
+                <Icon
+                  icon="mdi:check-decagram"
+                  style={{
+                    marginLeft: '0.5em',
+                    fontSize: '22px',
+                  }}
+                />
+              </Tooltip>
+            )}
+          </Box>
+        </Box>
+        <CardContent
+          style={{
+            margin: '1rem 0rem',
+            height: '15vh',
+            overflow: 'hidden',
+            paddingTop: 0,
+          }}
+        >
+          <Box
+            style={{
+              overflow: 'hidden',
+              textOverflow: 'ellipsis',
+              whiteSpace: 'nowrap',
+            }}
+          >
+            <Tooltip title={props.display_name} />
+            <Typography component="h5" variant="h5">
+              <HeadlampRouterLink
+                routeName="/plugin-catalog/:repoName/:pluginName"
+                params={{ repoName: props.repository?.name, pluginName: props.name }}
+              >
+                {props.display_name}
+              </HeadlampRouterLink>
+            </Typography>
+          </Box>
+          <Box display="flex" justifyContent="space-between" my={1}>
+            <Typography>v{props.version}</Typography>
+            <Box
+              marginLeft={1}
+              style={{
+                overflow: 'hidden',
+                textOverflow: 'ellipsis',
+                whiteSpace: 'nowrap',
+              }}
+            >
+              <Tooltip title={props?.repository?.name || ''}>
+                <Typography>{props?.repository?.name || ''}</Typography>
+              </Tooltip>
+            </Box>
+          </Box>
+          <Divider />
+          <Box mt={1}>
+            <Typography>
+              {props?.description?.slice(0, 100)}
+              {props?.description?.length > 100 && (
+                <Tooltip title={props?.description}>
+                  <Typography>…</Typography>
+                </Tooltip>
+              )}
+            </Typography>
+          </Box>
+        </CardContent>
+        <CardActions
+          style={{
+            justifyContent: 'space-between',
+            padding: '14px',
+          }}
+        >
+          <Link href={props?.repository?.url} target="_blank">
+            Learn More
+          </Link>
```

Comment:
There's currently no way to indicate from the list which plugins we have already installed or not. For now we could simply add an "Installed" label to the right side of the CardActions.

Reply:
Added a label